packages/common/src/types/certificate.ts

```diff
@@ -298,6 +301,23 @@ export interface ConfigUeCauseOfDeathList extends CertificateDataConfig {
   list: ConfigUeCauseOfDeathControl[]
 }
 
+export interface ConfigUeViewText extends CertificateDataConfig {
+  label?: string
+}
+
+export interface ConfigUeViewList extends CertificateDataConfig {
+  label?: string
```

Comment:
Should we call this text instead of label? Its what we normaly call this types of "headers" for questions.

Reply:
Text is already there in the interface it extends, this is on top of that and is used in some cases, but definitely not always. So use text

Reply:
Awsome!